features/ta_task_validate_image.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Verify Conforma Trusted Artifact Tekton Task
+  Verify Conforma Tekton Task feature scenarios
+
+  Background:
+    Given a cluster running
+    Given stub tuf running
+    Given stub git daemon running
+
+  Scenario: Golden container image with trusted artifacts
+    Given a working namespace
+    Given a snapshot artifact with content:
+      ```
+      {
+		    "components": [
+			    {
+			      "containerImage": "quay.io/hacbs-contract-demo/golden-container@sha256:e76a4ae9dd8a52a0d191fd34ca133af5b4f2609536d32200a4a40a09fdc93a0d"
+			    }
+		    ]
+		  }
+      ```
+    Given a cluster policy with content:
+      ```
+      {
+        "publicKey": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAERhr8Zj4dZW67zucg8fDr11M4lmRp\nzN6SIcIjkvH39siYg1DkCoa2h2xMUZ10ecbM3/ECqvBV55YwQ2rcIEa7XQ==\n-----END PUBLIC KEY-----",
+        "sources": [
+          {
+            "policy": [
+              "github.com/enterprise-contract/ec-policies//policy/release",
+              "github.com/enterprise-contract/ec-policies//policy/lib"
+            ],
+            "config": {
+              "include": [
+                "slsa_provenance_available"
+              ]
+            }
+          }
+        ]
+      }
+      ```
+    When version 0.1 of the task named "verify-conforma-konflux-ta" is run with parameters:
+      | SNAPSHOT_FILENAME       | snapshotartifact                                                                                    |
+      | SOURCE_DATA_ARTIFACT    | oci:${REGISTRY}/acceptance/snapshotartifact@${BUILD_SNAPSHOT_DIGEST} |
+      | POLICY_CONFIGURATION    | ${NAMESPACE}/${POLICY_NAME}                                                                              |
+      | STRICT                  | true                                                                                                     |
+      | IGNORE_REKOR            | true                                                                                                     |
+      | TRUSTED_ARTIFACTS_DEBUG | "true"                                                                                                   |
+      | ORAS_OPTIONS            | --plain-http                                                                                             |
+    Then the task should succeed
+     And the task logs for step "report" should match the snapshot
```

Comment:
I think the step is called "report-json", not "report", but I might be missing something.

Reply:
It is. I was missing something.